CIP-0000.md

```diff
@@ -0,0 +1,198 @@
+- - - -
+lang: en
+cip: 0000
+title: Celo Improvement Proposal (CIP) Process
+status: Draft
+type: Meta
+discussions-to:
+author: Yaz Khoury (@YazzyYaz), James Prestwich (@prestwich), Pranay Mohan (@pranaymohan), Mariano Cortesi (@mcortesi)
+created: 2020-09-22
+license: CC0 1.0
+- - - -
+
+# Abstract
+A Celo Improvement Proposal (CIP) describe standards for the Celo platform, including the core protocol specifications, SDK, and contract standards. It is a design document providing information to the Celo community, or describing a new feature for Celo or its processes or environment, a rationale for the proposal, detailed solution including technical specifications, and, if any, a list of potential risks. 
+
+The CIP proposer, the **author** is responsible for building consensus within the community and documenting whether there is rough consensus on their proposal. 
+
+Because the CIPs are maintained as text files in a versioned repository, their revision history is the historical record of the feature proposal.
+
+Participation in the Celo project and the CIP process is informed by the Celo [Community Tenets](https://celo.org/community) and subject to the  [Code of Conduct](https://celo.org/code-of-conduct).
+
+# CIP Types
+There are three types of CIP:
+
+* A **Standards Track CIP** describes any change that affects most or all Celo implementations, such as a change to the network protocol, a change in block or transaction validity rules, proposed application standards/conventions, or any change or addition that affects the interoperability of applications using Celo. Furthermore Standard CIPs can be broken down into the following categories. Standards Track CIPs consist of two parts, a design document and implementation.
+	* **Core** - improvements requiring a consensus fork, as well as changes that are not necessarily consensus critical but may be relevant to  “core dev” discussions, and the validator/node strategy.
+	* **Governance** - improvements to the governance proposal to the Celo platform including parameter changes. It should explain the reason for the change, enumerate the smart contract methods that will be called with parameters, and describe the impact and any risks. 
+	* **Networking** - improvements to networking protocol specifications such as devp2p, and proposed improvements like whisper and swarm.
+	* **Interface** - improvements around client [API/RPC] specifications and standards, and also certain language-level standards like method names and contract ABIs.
+	* **CRC** - (Celo [Request for Comments](https://en.wikipedia.org/wiki/Request_for_Comments)) application-level standards and conventions, including contract standards such as token standards, name registries, URI schemes, library/package formats, and wallet formats.
```

Comment:
I think it's good to have an advisory process alongside the CIP process here, but I don't think these are all good examples of the things we would want to make only advisory (see discussion below)